common/src/main/templates/io/netty/util/collection/KObjectHashMap.template

```diff
@@ -458,10 +475,113 @@ public class @K@ObjectHashMap<V> implements @K@ObjectMap<V>, Iterable<@K@ObjectM
         }
```

Comment:
@nmittler - Sorry I could not leave a comment on the line but I was talking about the `toInternal` call in the `rehash(..)` method.